source/includes/available-permissions.rst

```diff
@@ -1,80 +1,326 @@
+        - :base_url:`Disable user <_static/api/studio.html#tag/users/operation/disableUser>`
+        - :base_url:`Reset user password <_static/api/studio.html#tag/users/operation/resetUserPassword>`
+    * - rebuild_database
+      - Project
+      - User is permitted to rebuild Crafter Studio’s database and object state with the underlying repository
+    * - remove_plugins
+      - Project
+      - User is permitted to :ref:`remove installed plugins <plugin-management>` |br|
+        Required when using the following APIs:
+
+        - :base_url:`Remove plugin <_static/api/studio.html#tag/marketplace/operation/removePlugin>`
+        - :base_url:`Plugin usage <_static/api/studio.html#tag/marketplace/operation/pluginUsage>`
+    * - search_plugins
+      - Project
+      - User is permitted to :ref:`search for plugins <plugin-management>`
+    * - set_item_states
+      - Project
+      - User is permitted to set item states |br|
+        Required when using the following APIs:
+
+        - :base_url:`Set item states <_static/api/studio.html#tag/workflow/operation/setItemStates>`
+        - :base_url:`Update item states <_static/api/studio.html#tag/workflow/operation/updateItemStates>`
+    * - site_status
+      - Project
+      - User is permitted to :ref:`get status of repository for a project <project-tools-git>` |br|
+        Required when using the API :base_url:`Repository status <_static/api/studio.html#tag/repository/operation/repositoryStatus>`
+    * - unlock_repository
+      - Project
+      - User is permitted to unlock the repository
+    * - update_cluster |enterpriseOnly|
+      - System
+      - User is permitted to update cluster
+    * - read_cluster |enterpriseOnly|
+      - System
+      - User is permitted to read cluster |br|
+        Required when using the following APIs:
+
+        - :base_url:`Get the local Studio instance cluster mode <_static/api/studio.html#tag/cluster/operation/getClusterMode>`
+        - :base_url:`Get all the members of the cluster <_static/api/studio.html#tag/cluster/operation/getClusterMembers>`
+    * - create_cluster |enterpriseOnly|
+      - System
+      - User is permitted to create cluster
+    * - delete_cluster |enterpriseOnly|
+      - System
+      - User is permitted to delete clusters
+    * - read_configuration
+      - Project
+      - User is permitted to read configuration content for project |br|
+        Required when using the following APIs:
+
+        - :base_url:`Get all model definitions for site <_static/api/studio.html#tag/model/operation/getAllModelDefinitions>`
+        - :base_url:`Get configuration content for site and configuration location <_static/api/studio.html#tag/configuration/operation/getConfiguration>`
+        - :base_url:`Get configuration history for site and configuration location <_static/api/studio.html#tag/configuration/operation/getConfigurationHistory>`
+    * - write_configuration
```

Comment:
write_configuration scope would be project
For system there is write_global_configuration